CWE-94-Python_darwin/GAIR-NLP_PC-Agent/fs.py

```
@@ -0,0 +1,78 @@
+    if os.path.exists(file_path):
+        try:
+            os.remove(file_path)
+        except OSError as e:
+            print(f"Failed to delete {file_path}: {e}")
+
+
+def delete_folder(folder_path):
+    try:
+        if os.name == 'nt':
+            # On Windows, use system command to delete
+            subprocess.call(f'rd /s /q "{folder_path}"', shell=True)
```

Ruff (0.13.2)

`subprocess` call with `shell=True` identified, security issue

(S602)